extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfigPersistenceUnit.java

```diff
@@ -112,6 +112,15 @@
     @ConfigItem
     public Optional<String> implicitNamingStrategy;
 
+    /**
+     * XML files to configure the entity mapping, e.g. {@code META-INF/my-orm.xml}.
+     * <p>
+     * Defaults to `META-INF/orm.xml` if it exists.
+     * Pass `no-file` to force Hibernate ORM to ignore `META-INF/orm.xml`.
+     */
+    @ConfigItem(defaultValueDocumentation = "META-INF/orm.xml if it exists; no-file otherwise")
+    public Optional<Set<String>> mappingFiles;
```

Comment:
Do we hot reload these files?

Reply:
I have absolutely no idea. I didn't do anything special to have them hot-reloaded, anyway.

They are processed in a build step: `io.quarkus.hibernate.orm.deployment.HibernateOrmProcessor#defineJpaEntities`.

I will add a test regarding hot-reload of XML mapping files. If it doesn't work, any hint on how I can tell Quarkus that some files are relevant to hot-reload?

Reply:
Looking at the PR, they are not enabled for hot reload. To do so, you might want to produce a `HotDeploymentWatchedFileBuildItem` just like https://github.com/quarkusio/quarkus/blob/ac05c00115efc9008e743ce6036a6d435687f5de/extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java#L200 or https://github.com/quarkusio/quarkus/blob/ac05c00115efc9008e743ce6036a6d435687f5de/extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java#L931

Reply:
Done! Thanks for the tip.

Reply:
Awesome, thanks. You are welcome.